src/rustllvm/PassWrapper.cpp

```diff
@@ -97,6 +97,55 @@ LLVMRustAddPass(LLVMPassManagerRef PM, Pass *pass) {
     pm->add(pass);
 }
 
+#define GEN_SUBTARGETS \
+    SUBTARGET(X86) \
+    SUBTARGET(ARM) \
+    SUBTARGET(AArch64) \
+    SUBTARGET(Mips) \
+    SUBTARGET(PPC)
```

Comment:
The LLVM components detected at build time are available through `cfg!(llvm_component)` in Rust, but I could not find anything similar for the C++ libraries. Is it ok to duplicate the detection code from `mk/llvm.mk`?